polkadot/xcm/xcm-executor/src/traits/asset_transfer.rs

```diff
@@ -23,9 +23,8 @@ use xcm::prelude::*;
 /// Errors related to determining asset transfer support.
 #[derive(Copy, Clone, Encode, Decode, Eq, PartialEq, Debug, TypeInfo)]
 pub enum Error {
-	/// Invalid non-concrete asset.
-	NotConcrete,
 	/// Reserve chain could not be determined for assets.
+	#[codec(index = 1)]
```

Comment:
No need to put this

Reply:
Removed in the commit: https://github.com/paritytech/polkadot-sdk/pull/3867/commits/819dae58dc8946802903bd65800bd6b31c32fc26